### PR TITLE
add some mtr case

### PR DESCRIPTION
**Pull request by lyq10085**

### CODE REVIEW

File: testcase/t/alias.result [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+(select t2.* from t2)
+99
+99
+select a, (select t2.* from t2) from t1;
+a	(select t2.* from t2)
+1	99
+11	99
+select t1.*, (select t2.* from t2) from t1;
+a	b	c	(select t2.* from t2)
+1	2	3	99
```
Comment (earayu) [inline testcase/t/alias.result]: why adding result files in t folder?
Is this just for the purpose of comparing with recorded result files? and you will delete them finally?

Comment (lyq10085) [reply testcase/t/alias.result]: I don't even remember I have done this intentionally. It's a mistake,I think.

Comment (earayu) [reply testcase/t/alias.result]: OK, don't forget to delete these result file in t folder.

**Review by earayu — APPROVED** [review]
LGTM

**Pull request merged**